client/types/disputes.d.ts

```diff
@@ -82,7 +82,7 @@ interface UploadFieldObject {
 	label: string;
 }
 
-export interface DisputeFileUpload {
+export interface FileUploadControlProps {
```

Comment:
Not sure if this is a good idea. @dmvrtx, @anu-rock, @KarlisJ WDYT?

Reply:
Renaming of this interface came up during a [slack discussion](p1664441804947769/1664438225.993279-slack-C02C00HCJ8J) where it was suggested to rename this interface to make it consistent with the [`FileUploadControl`](https://github.com/Automattic/woocommerce-payments/blob/54cfd573047adf349e074264ca2d967e4b3ade50/client/components/file-upload/index.tsx#L22) component, that uses this interface. 

I took care to check all references of `DisputeFileUpload` and rename it to `FileUploadControlProps`

Reply:
The link to slack discussion is broken for me. But after spending some time understanding the changes, I'm ok with the decision to rename the interface.

Reply:
Thanks, Karlis! I realized that if I copy links of threaded replies in slack, it does not resolve. I have replaced it with the link to the main message of the thread ( for future reference )

Reply:
I am marking this as resolved for now. Happy to revisit if needed.

Reply:
Marking this as resolved for now. Happy to revisit if needed.